app/views/users/folders/binaries/show.html.erb

```diff
@@ -4,9 +4,12 @@
 <%= link_to users_like_file_path(@binary), method: :post, class: 'btn btn-warning' do %><span class="glyphicon glyphicon-thumbs-up">
 </span> Like File <% end %>
 
-<div class="likes">
-  <p>File Likes: <%= @binary.likes.count %> </p>
-</div>
+<span class="badge badge-pill badge-danger">
+  <div class="likes">
+    File Likes: <%= @binary.likes.count %>
```

Comment:
i suggest writing a method that encapsulates the like count - so you can write `@binary.like_count` - the view doesn't need to know about needing to access  a binary's likes then count them